src/state_machine.zig

```diff
@@ -709,7 +702,7 @@ pub fn StateMachineType(
             self: *const StateMachine,
             client_release: vsr.Release,
             operation: Operation,
-            input: []align(16) const u8,
+            input: []const u8,
```

Comment:
Vopr seed: `./zig/zig build -Drelease vopr -- --lite 12293026746782977520 `

Cause: Vopr workload generating too many batched events.

Root cause: `StateMachine.input_valid` is not enough to validate a batched request. It validates each batch isolated, but we must also validate all batches combined and check if they fit into the commit limits.

I :heart: VOPR!